src/main/resources/templates/progress/teamDetail.html

```diff
@@ -7,8 +7,7 @@
 <div class="container">
     <div class="row">
         <div class="col-sm-12">
-            <h1>Tým</h1>
-            <p class="subheading" th:text="${team.name}"></p>
+            <h2 th:text="${team.name}"></h2>
```

Comment:
Tuto zmenu bych nedelal, protoze kdyz ten nazev je divny, pak nevis, co na te strance je. Ale Je to jen muj nazor, takze necham i na ostatnich.

Reply:
Pokud se ti to nezdá, tak to nechme jak to bylo pro jistotu :)

Reply:
revertnuto :)